Core/clim-core/panes.lisp

```diff
@@ -243,17 +243,20 @@ order to produce a double-click")
               max-height))))
 
 (defun make-space-requirement (&key (min-width 0) (min-height 0)
-                                    (width min-width) (height min-height)
-				    (max-width +fill+) (max-height +fill+))
-  (assert (<= 0 min-width width max-width) (min-width width max-width))
-  (assert (<= 0 min-height height max-height) (min-height height max-height))
+                                 (width min-width) (height min-height)
+                                 (max-width +fill+) (max-height +fill+))
+  ;; Defensive programming. space-requierment-+ may double max-width=+fill+.
```

Comment:
Typo in "requirement"; I didn't understand the rest of the sentence either.

Reply:
when we call `space-requirement+` on two space requirements which has `max-width` `+fill+`, then we end up with `max-width` being `(* +fill+ +fill+)`, hence *bigger* than our maximum size.